bin/detab.el

```diff
@@ -0,0 +1,5 @@
+
+(defun exorcize-tabs ()
+  (interactive)
+  (message "Converting tabs to spaces in buffer: `%s'" (buffer-name))
+  (untabify (point-min) (point-max)))
```

Comment:
Also nice

Reply:
Hah cheers! Feel free to use it to keep finishing them off as you visit files!